HomeTask1/Calculator/Calculations/ConsoleInput.cs

```diff
@@ -0,0 +1,44 @@
+namespace HomeTask1.Calculator.Calculations;
+
+public class ConsoleInput
+{
+    public static float GetFirstNumberFromConsole()
+    {
+        Console.Write("Enter number_1: ");
+        if (float.TryParse(Console.ReadLine(), out var number1)) return number1;
```

Comment:
ИМХО, так лучше не делать, у тебя то и сейчас конструкция if достаточно сложная, а добавь сюда еще 3 параметра и мозг сломается где тело выполняемого if. Луче вот так

```suggestion
        if (float.TryParse(Console.ReadLine(), out var number1)) 
        {
            return number1;
        }
```

Reply:
это мне так райдер посоветовал)) я думала так модно

Reply:
Это у тебя так настройки сконфигурированы

Reply:
нам надо это изучить